### ISSUE TITLE
Normalizing through the Learner only normalizes the training set

### ISSUE THREAD

**Issue by BobMcDear** [fastai/fastai#3281]
Please confirm you have the latest versions of fastai, fastcore, and nbdev prior to reporting a bug (delete one): YES

**Describe the bug**
When passing ```normalize=True``` to a ```cnn_learner``` and not normalizing the DataLoaders, fastai doesn't normalize the validation set.

**To Reproduce**
Please have a look at [this](https://colab.research.google.com/drive/17uCSWciimk8RHMN1Zxcqlpb4PFMj8wXA?usp=sharing) Colab notebook.

**Expected behavior**
fastai should normalize both datasets and not just the training set.

**Additional context**
There's an [issue](https://github.com/fastai/fastai/issues/3234) with ```Learner.show_results``` that causes images to be gray, which is a result of this bug.

**Comment by BobMcDear** [comment]
This is completely my mistake. I assumed the latest version of fastai is on pip, and this issue goes away if fastai is installed via Github.

**Comment by riven314** [comment]
I encountered the same issue when using fastai in kaggle kernel (2.2.7)
Would you mind pointing me to the PR that fixed the issue?
I would like to know what cause the issue, thanks!

**Comment by BobMcDear** [comment]
Of course, [here](https://github.com/fastai/fastai/pull/3268) it is.

**Comment by riven314** [comment]
@BobMcDear 
Thanks so much for the pointer!

**Comment by tcapelle** [comment]
Please @jph00 make a release.

**Comment by jph00** [comment]
@tcapelle 2.3.1 was released 2 weeks ago.
